docs/source/index.rst

```diff
@@ -41,34 +55,33 @@ The following commands will download and build UCX v1.6 :doc:`release <download>
     $ ./contrib/configure-release --prefix=$PWD/install
     $ make -j8 install
 
+    $ gcc test/examples/ucp_client_server.c -lucp -lucs -o ucp_client_server \
+          -Iinstall/include -Linstall/lib
+    $ export LD_LIBRARY_PATH=$PWD/instal/lib
+    $ ./ucp_client_server &
+    $ ./ucp_client_server -a 2.1.3.20
+    ...
+    ----- UCP TEST SUCCESS -------
+
+    UCX Client-Server Hello World
+
+    ------------------------------
+
 
 Documentation
 *************
 
-* TODO API
+*  API doc: `HTML <api/v1.6/html>`_ `PDF <api/v1.6/ucx.pdf>`_
 * `Examples <https://github.com/openucx/ucx/tree/v1.6.x/test/examples>`_
 
 
 Projects using UCX
 ******************
 
+* `UCX-PY <https://ucx-py.readthedocs.io/en/latest>`_
+* `Dask <https://blog.dask.org/2019/06/09/ucx-dgx>`_
+* `SparkUCX <http://github.com/openucx/sparkucx>`_
+* `NCCL <https://github.com/Sergei-Lebedev/nccl/tree/ucx>`_
```

Comment:
nccl is upstream path

Reply:
updated, though i didn't find UCX in NCCL upstream code